### PR TITLE
dpg, cadl-ranch, add tests for property-optional

### PR DESCRIPTION
**Pull request by haolingdong-msft**
1. add tests for properties-optional
2. update existing tests: add back disabled tests and remove unused imports.

### CODE REVIEW

File: cadl-tests/src/test/java/com/models/property/optional/DurationOperationClientTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// Licensed under the MIT License.
+
+package com.models.property.optional;
+
+import com.models.property.optional.models.DurationProperty;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("cadl-ranch definition error")
```
Comment (haolingdong-msft) [inline cadl-tests/src/test/java/com/models/property/optional/DurationOperationClientTest.java:11]: disabled these tests due the a bug in cadl-ranch, I've created a fix pr in cadl-ranch: https://github.com/Azure/cadl-ranch/pull/113

File: cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.authentication.apikey;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuthenticationApiKeyClientTest {
```
Comment (weidongxu-microsoft) [inline cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java:7]: I should have added test for auth?

**Review by weidongxu-microsoft — APPROVED** [review]

File: cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java (hunk 2 above)
Comment (haolingdong-msft) [reply cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java:7]:
```  
"Authentication_ApiKey_valid": "pass",
  "Authentication_ApiKey_invalid": "fail",
  "Authentication_OAuth2_valid": "pass",
  "Authentication_OAuth2_invalid": "fail",
```

Auth tests seems failed, so I'm still working on figuring out the reason

Comment (weidongxu-microsoft) [reply cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java:7]: ignore for now, the 2 API is design to fail as invalid credential.

Comment (haolingdong-msft) [reply cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java:7]: ok, I checked the logs, the invalid calls are called, maybe `fail` is expected, I will delete the file and ignore those two `fail`

Comment (weidongxu-microsoft) [reply cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java]: said it is expected, the call is designed to return 401/403

please prioritize on those does not have test yet.

Comment (haolingdong-msft) [reply cadl-tests/src/test/java/com/authentication/apikey/AuthenticationApiKeyClientTest.java]: I'm working on other cases now.

**Pull request merged**